<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameNV</class>
 <widget class="QFrame" name="FrameNV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>111</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Mã Nhân Viên</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Tên Nhân Viên</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Phòng Ban</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Chức Vụ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Giới Tính</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Ngày Sinh</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_10">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>160</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <widget class="QPushButton" name="btChoose">
      <property name="text">
       <string>Chọn file</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>70</y>
     <width>331</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="txtIDNV"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txtfullName"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboPB"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboCV"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboSex"/>
    </item>
    <item>
     <widget class="QDateEdit" name="deDateOfBirth"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>190</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLineEdit" name="txtSDT"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txtCMT"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txtTrinhDo"/>
    </item>
    <item>
     <widget class="QLineEdit" name="txtHSL"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="dataTab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>1191</width>
     <height>141</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Mã Nhân Viên</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tên Nhân Viên</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Phòng Ban</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Chức Vụ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Giới Tính</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ngày Sinh</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SDT</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>CMT</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Trình Độ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Hệ Số Lương</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>70</y>
     <width>81</width>
     <height>199</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QPushButton" name="btAdd">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Thêm</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btFix">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sửa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btDel">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Xóa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btClear">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>190</y>
     <width>101</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Số Điện Thoại</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Số CMT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Trình Độ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Hệ Số Lương</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NHÂN VIÊN</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_9">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>59</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QLabel" name="lbImg">
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>90</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Ảnh</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
